src/Components/AddButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {View, TouchableOpacity, Text, StyleSheet} from 'react-native';
+
+type props = {
+  setShowCrypto: (value: boolean) => void;
+};
+
+export default function AddButton({setShowCrypto}: props) {
```

Comment:
use arrow function and implicit return. 
when you do that move the export default to the end of the file